--- a/pulseaudio-ctl	2013-10-12 22:40:29.000000000 +0200
+++ b/pulseaudio-ctl	2013-10-27 15:22:35.290272111 +0100
@@ -60,6 +60,16 @@ case "$1" in
     [[ "$CURVOL" -le 0 ]] && exit 0 ||
       pactl set-sink-volume "$SINK" -- -5%
     ;;
+  F|f|[F,f]ull)
+    # adjust volume to 100%
+    setup
+    pactl set-sink-volume "$SINK" -- 100%
+    ;;
+  N|n|[N,n]ormal)
+    # adjust volume to 70%
+    setup
+    pactl set-sink-volume "$SINK" -- 70%
+    ;;
   M|m|[M,m]u|[M,m]ute)
     # mutes the volume entirely
     setup
@@ -67,9 +77,11 @@ case "$1" in
     ;;
   *)
     setup
-    echo -e " ${BLD}$0 ${NRM}${BLU}{up,down,mute}${NRM}"
+    echo -e " ${BLD}$0 ${NRM}${BLU}{up,down,full,normal,mute}${NRM}"
     echo
     echo -e " ${BLD}Where ${NRM}${BLU}up${NRM}${BLD} and ${NRM}${BLU}down${NRM}${BLD} adjust volume in ±5 % increments.${NRM}"
+    echo -e " ${BLD}Where ${NRM}${BLU}full${NRM}${BLD} adjust volume to 100%${NRM}"
+    echo -e " ${BLD}Where ${NRM}${BLU}normal${NRM}${BLD} adjust volume to 70%${NRM}"
     echo -e " ${BLD}Where ${NRM}${BLU}mute${NRM}${BLD} toggles the mute status on/off.${NRM}"
     echo
     echo -e " ${BLD}Volume level  : ${NRM}${RED}$CURVOL %${NRM}"
