--- a/pulseaudio-ctl	2013-10-12 22:40:29.000000000 +0200
+++ b/pulseaudio-ctl	2013-10-27 13:34:41.960193392 +0100
@@ -54,6 +54,11 @@ case "$1" in
     [[ "$CURVOL" -ge 100 ]] && exit 0 ||
       pactl set-sink-volume "$SINK" -- +5%
     ;;
+  F|f|[F,f]ull)
+    # raise volume by 100%
+    setup
+    pactl set-sink-volume "$SINK" -- +100%
+    ;;
   D|d|[D,d]own|[D,d]o)
     # lowers volume by 5%
     setup
